go/nbs/table.go

```diff
@@ -117,6 +118,13 @@ const (
 	maxChunkSize       uint64 = 0xffffffff // Snappy won't compress slices bigger than this
 )
 
+var crcTable = crc32.MakeTable(crc32.Castagnoli)
+
+// CORRECT?
```

Comment:
assuming so. i put this here because the snappy library does some weird bit shifting of the result (for internal use).